kview/src/main/java/dev/ikm/komet/kview/mvvm/view/genediting/SemanticFieldsController.java

```diff
@@ -110,23 +111,28 @@ private void setupSemanticDetailsUI(Latest<SemanticEntityVersion> semanticEntity
                 node = klField.klWidget();
                 node.setUserData(klField.field());
             } else if (dataTypeNid == TinkarTerm.STRING_FIELD.nid() || fieldRecord.dataType().nid() == TinkarTerm.STRING.nid()) {
-
                 KlStringFieldFactory stringFieldTextFactory = new KlStringFieldFactory();
                 ObservableSemantic observableSemantic = ObservableEntity.get(semanticEntityVersionLatest.get().nid());
                 ObservableSemanticSnapshot observableSemanticSnapshot = observableSemantic.getSnapshot(getViewProperties().calculator());
                 ImmutableList<ObservableField> observableFields = observableSemanticSnapshot.getLatestFields().get();
                 ObservableViewBase observableViewBase = getViewProperties().nodeView();
-
                 ObservableField<String> stringObservableField =observableFields.get(fieldRecord.fieldIndex());
-
                 //StringField klWidget returns the widget container which is an HBox with a hard coded label
                 node = stringFieldTextFactory.create(stringObservableField, observableViewBase, true).klWidget();
-
                 node.setUserData(stringObservableField);
             } else if (dataTypeNid == TinkarTerm.COMPONENT_ID_SET_FIELD.nid()) {
                 node = rowf.createReadOnlyComponentSet(getViewProperties(), fieldRecord);
             } else if (dataTypeNid == TinkarTerm.DITREE_FIELD.nid()) {
                 node = rowf.createReadOnlyDiTree(getViewProperties(), fieldRecord);
+            }else if (dataTypeNid == TinkarTerm.FLOAT_FIELD.nid() || fieldRecord.dataType().nid() == TinkarTerm.FLOAT.nid()) {
```

Comment:
@swaroopsalvi @aks8m 
Can we confirm the check of `TinkarTerm.FLOAT`? because a pattern should only use FLOAT_FIELD. I believe FLOAT is a different concept and used for something else.